### ISSUE TITLE
Indonesia GTM - Team 1: Webmaker Content

### ISSUE THREAD

**Issue by RizRiyz** [mozilla/webmaker-launch#44]
Goal : 
- Create 1000 projects in 60 days post launch

Team member : @yofiesetiawan, @nhie, @deryanEM
Team pilot : @deryanEM

Document Plan : https://docs.google.com/document/d/1JZFnWXNrbtAjPMK0Ig4wI-aozbdI12zWUyjw3q2UaUw/edit?usp=sharing

**Comment by secretrobotron** [comment]
Some more content ideas: https://etherpad.mozilla.org/8LtQGBkY6A

_Note: we should add these to the content dashboard: https://docs.google.com/spreadsheets/d/1Ab53yEZKTlXw47ISP1HewAcL1kOwyczsZDy9ueogURY/edit#gid=0_

**Comment by yofiesetiawan** [comment]
Timeline
1. Gather list of schools and campus we can visit
2. Create awesome local contents for ideas
3. Run events on school and campus visits, and also promoting our big launch event
4. Run big launch event

What we need:
1. Proposal Letter format for schools and campus
2. Swags

Content Ideas:
1. Comic - build partnership with communities like comic illustrators
2. Personality Test
3. Scrapbook - familiar within girls in Indonesia - https://beta.webmaker.org/users/50713/projects/204
4. Sports - use sports that is familiar for indonesian people
5. Name Card or Profile
6. Event Promotion

**Comment by secretrobotron** [comment]
@yofiesetiawan @deryanEM are you hoping to build partnerships with illustrators by visiting campuses? Or do you have another plan for approaching them?

**Comment by yofiesetiawan** [comment]
@secretrobotron ina has relationships with illustrator community, we will try to first approach them. while visiting campuses and schools is different, we will need to prepare a letter for schools and campuses. We hope that you don't mind to help us sign on the letter..

**Comment by secretrobotron** [comment]
Not at all, @yofiesetiawan! Ina said she had a template for me to work with. If you've got it, can you post it here or give me access? I'm sure she did send it to me last week, but it's buried in email.

**Comment by mauredtha** [comment]
Halo @secretrobotron, Ina said you can check the format template in team's 2 github , this is the link :

Team 2 : https://github.com/mozilla/webmaker-launch/issues/45

**Comment by mauredtha** [comment]
https://docs.google.com/document/d/1pCW4ss5Yy-SymY5bSCUEwcutYh2OZBfpe_5avlRE3UU/edit?usp=sharing

**Comment by secretrobotron** [comment]
That's what I needed! Thanks :).

**Comment by mauredtha** [comment]
@secretrobotron pleasure :)

**Comment by mauredtha** [comment]
We have created some webmaker content for helping Team 2  #45 to make presentation and tutorial for event on school visitation : 
https://beta.webmaker.org/users/50713/projects/3496
https://beta.webmaker.org/users/50713/projects/3448
https://beta.webmaker.org/users/50713/projects/3445
https://beta.webmaker.org/users/50713/projects/204
https://beta.webmaker.org/users/426154/projects/2926
https://beta.webmaker.org/#/project?user=438321&project=2369
https://beta.webmaker.org/users/111027/projects/3469
https://beta.webmaker.org/#/project?user=426154&project=2928

**Comment by mauredtha** [comment]
Hey @secretrobotron , sorry for slow update, this is the list of the content's link
https://docs.google.com/spreadsheets/d/147Cq03evgpTSC7E0_Svy5Y-32N85273sMKsAsZ2xwBM/edit?usp=sharing

**Comment by secretrobotron** [comment]
Thanks @deryanEM :smile_cat: Added to https://github.com/mozilla/webmaker-launch/wiki/Indonesia#content

**Comment by KevZawacki** [comment]
Hi @deryanEM, @secretrobotron made a neat tool for finding and sorting local Webmaker projects:  http://secretrobotron.github.io/webmaker-launch/new-projects.html

It can help to put together a small list of cool projects every week.

**Comment by mauredtha** [comment]
Wow !! Thanks a lot @KevZawacki , it's very cool and very helpful for our team . @secretrobotron

**Comment by mauredtha** [comment]
Halo @secretrobotron , i have a question. I was opened the tools for  finding and sorting local webmaker project http://secretrobotron.github.io/webmaker-launch/new-projects.html , but why there are so many empty page like this page http://secretrobotron.github.io/webmaker-launch/new-projects.html?language=id-ID&page=50 and http://secretrobotron.github.io/webmaker-launch/new-projects.html?language=id-ID&page=51 ? but sometimes when click next button, the project show, or perhaps it is the last page of all the data?

**Comment by KevZawacki** [comment]
Hi @deryanEM, sorry you're having this issue. 

This tool shows _all_ Webmaker projects, whether they are finished or have only just been started. That may be part of the problem. 

Feel free to click "next" through the empty pages until projects start appearing again.

**Comment by mauredtha** [comment]
I thought  it is the last page of all the data, okay no problem i am understand now. thank you @KevZawacki :)
